src/reaper_osara.cpp

```diff
@@ -3583,6 +3583,25 @@ void cmdUnselAllTracksItemsPoints(Command* command) {
 		outputMessage(translate("unselected tracks/items/envelope points"));
 }
 
+void reportTempoTimeSig() {
+	ReaProject* proj=EnumProjects(-1, nullptr, 0);
+	if(proj==nullptr) return;
```

Comment:
nit: Brace if body to avoid ambiguity, boolean check rather than comparing against nullptr.

```suggestion
	if (!proj) {
		return;
	}
```